PowerShellTools/Intellisense/IntelliSenseManager.cs

```diff
@@ -119,7 +119,14 @@ public int Exec(ref Guid pguidCmdGroup, uint nCmdId, uint nCmdexecopt, IntPtr pv
                 return VSConstants.S_OK;
             }
 
-            //pass along the command so the char is added to the buffer 
+            // Check the char at caret before pass along the command
+            // If command is backspace and completion session is active, then we need to see if the char to be deleted is an IntelliSense triggering char
+            // If yes, then after deleting the char, we also dismiss the completion session
+            // Otherwise, just filter the completion lists
+            ITrackingPoint caretCharPosition = _textView.TextSnapshot.CreateTrackingPoint(_textView.Caret.Position.BufferPosition.Position - 1, PointTrackingMode.Positive);

```

Comment:
Quick sanity check: Are we good if this is done at the very beginning of the file?  eg, what if our Caret is now @ pos 0?